react-native-aztec/android/build.gradle

```diff
@@ -12,7 +12,7 @@ buildscript {
         wordpressUtilsVersion = '1.22'
         espressoVersion = '3.0.1'
 
-        aztecVersion = 'v1.3.28'
+        aztecVersion = '0fa41a08efef6092a6bd8c7b31e948cb21144d02'
```

Comment:
Just noticed that in the [AztecEditor-Android PR](https://github.com/wordpress-mobile/AztecEditor-Android/pull/841/files) we're updated the Aztec version from `1.3.28` to `1.3.29`. I'm curious why we don't want the same thing here?

Reply:
Hey @mchowning my idea was to not publish (aztec) release until we don't get approve on this PR.
 
As if we find some bug during testing `gb-mobile` or `wpandroid` (against Aztec release branch) it seems that it would be easier to do the fix on Aztec develop without publishing a new hotfix or so. Does this makes a sense?

I have left a comment in the description of this task which will remind me to change hash with `v1.3.29` tag. when everything is ready.